src/app/pages/overlays/basic-bottom-sheet/basic-bottom-sheet.component.html

```diff
@@ -0,0 +1,25 @@
+<mat-toolbar color="primary">
+    <button *ngIf="isSmall" style="margin-left: -8px;" mat-icon-button (click)="toggleMenu()">
+        <mat-icon>menu</mat-icon>
+    </button>
+    <div>Basic Bottom Sheet</div>
+</mat-toolbar>
+<mat-nav-list *ngIf="data.length > 0">
+    <pxb-info-list-item
+        *ngFor="let i of data; let index = index"
+        [class.active]="i.active"
+        (click)="showBottomSheet()"
```

Comment:
There is supposed to be an icon in the top toolbar that opens the bottomsheet (global actions) rather than on each individual row.